### ISSUE TITLE
[BUG?] Error saving theme to file

### ISSUE THREAD

**Issue by boromyr** [Abdelrhman-AK/WinPaletter#83]
## **Describe the bug**
When I try to save my settings to a file with version 1.0.7.1 I get an error message whose log file is attached below.
## **WinPaletter Log**
```vbnet
'General Information
'-----------------------------------------------------------
Report.Date = "martedì 28 febbraio 2023 17:37:51"
OS = "Microsoft Windows 11 Pro - 10.0.22623.1325 - 64-bit"
WinPaletter.Version = "1.0.7.1"

'Error Details
'-----------------------------------------------------------
Exception.Message = "Riferimento a un oggetto non impostato su un'istanza di oggetto."
Exception.StackTrace = {
   in WinPaletter.WinTerminal.Save(String File, Mode Mode, Version Version)
   in WinPaletter.WinTerminal.ToString(String Signature, Version Edition)
   in WinPaletter.CP.ToString()
   in WinPaletter.CP.Save(CP_Type SaveTo, String FileLocation, TreeView TreeView)
   in WinPaletter.MainFrm.XenonButton7_Click(Object sender, EventArgs e)
   in System.Windows.Forms.Control.OnClick(EventArgs e)
   in System.Windows.Forms.Button.OnClick(EventArgs e)
   in System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   in WinPaletter.XenonButton.OnMouseUp(MouseEventArgs e)
   in System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   in System.Windows.Forms.Control.WndProc(Message& m)
   in System.Windows.Forms.ButtonBase.WndProc(Message& m)
   in System.Windows.Forms.Button.WndProc(Message& m)
   in System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   in System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   in System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
         }
Exception.Data = "There is no included data in Exception"
Exception.Target.Sub_Function = "Save @ WinPaletter"
Exception.Assembly = "WinPaletter, Version=1.0.7.1, Culture=neutral, PublicKeyToken=null"
Exception.Assembly.File = "C:\Program Files\WinPaletter.exe"
Exception.HRESULT = "-2147467261"
```

## **Screenshots**
![image](https://user-images.githubusercontent.com/52782032/221919602-a61df837-7434-4c22-a128-0f3433bc38c0.png)

## **Additional context**
The error only occurs when I try to overwrite an already existing file. I loaded the current settings, created a new file on the desktop and tried overwriting, but the error still occurs.

WP v1.0.7.1
Windows 11 22H2
Saved theme: [Dark.wpth.txt](https://github.com/Abdelrhman-AK/WinPaletter/files/10852701/Dark.wpth.txt)

Off Topic: Unable to compile from source as files in:
```
<data name="ActiveApp_Taskbar" type="System.Resources.ResXFileRef, System.Windows.Forms">
  <value>..\..\..\..\VB Working Area - Projects\WinPaletter_WorkingArea\Windows Elements\ActiveApp_Taskbar.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
</data>
```

**Comment by Abdelrhman-AK** [comment]
- I'll fix saving issue.
- The problem of compiling is due to non-existing resources files in repos. I'll fix this issue too.

**Comment by boromyr** [comment]
Maybe the problem is related to this, but I can't load the font currently in use by the system, and also the one set in the .wpth file.
Only tested with Segoe UI Variable

**Comment by Abdelrhman-AK** [comment]
The problem is specifically inside code that saves the preferences of Windows Terminals to text.
And so I don't think it is a font issue, but I'll try to find what causes this and fix it soon.

**Comment by boromyr** [comment]
The specified problem has been solved with 1.0.7.2, also fails to read or set the settings correctly in my dark mode file, do I open a new topic with the observed problems? Currently, I'm back to 1.0.7.0

**Comment by Abdelrhman-AK** [comment]
Open a new problem
